tools/accuracy_checker/configs/single-human-pose-estimation-0001.yml

```diff
@@ -0,0 +1,29 @@
+models:
+  - name: single-human-pose-estimation-0001
+
+    launchers:
+      - framework: dlsdk
+         tags:
+          - FP32
+        model:   public/single-human-pose-estimation-0001/FP32/single-human-pose-estimation-0001.xml
+        weights: public/single-human-pose-estimation-0001/FP32/single-human-pose-estimation-0001.bin
+        adapter:
+          type: single_human_pose_estimation
+      - framework: dlsdk
+         tags:
+          - FP16
+        model:   public/single-human-pose-estimation-0001/FP16/single-human-pose-estimation-0001.xml
+        weights: public/single-human-pose-estimation-0001/FP16/single-human-pose-estimation-0001.bin
+        adapter:
+          type: single_human_pose_estimation
+
+    datasets:
+      - name: ms_coco_single_keypoints
+        preprocessing:
+          - type: transformed_crop_with_auto_scale
+            dst_height: 384
+            dst_width: 288
+            stride: 8
+        metrics:
+          - name: AP
+            type: coco_orig_keypoints_precision
```

Comment:
please add `global_definitions: ../dataset_definitions.yml` at the end like in other configs e.g. [human-pose-estimation-0001](https://github.com/opencv/open_model_zoo/blob/develop/tools/accuracy_checker/configs/human-pose-estimation-0001.yml#L62)